### PR TITLE
SSL: Document SSL_add_{file,dir,store}_cert_subjects_to_stack()

### PR DESCRIPTION
**Pull request by levitte**
This also removes the incorrect documentation comments by those
functions, and fixes a bug in SSL_add_store_cert_subjects_to_stack().

### CODE REVIEW

File: util/missingssl111.txt [hunk 1]
```diff
@@ -28,8 +28,6 @@ SSL_CTX_set_srp_verify_param_callback
 SSL_CTX_set_trust
 SSL_SRP_CTX_free
 SSL_SRP_CTX_init
-SSL_add_dir_cert_subjects_to_stack
-SSL_add_file_cert_subjects_to_stack
```
Comment (levitte) [inline util/missingssl111.txt]: Because #10403

**Review by t8m — APPROVED** [review]
LGTM

Could you please amend the commit message to describe the bug being fixed.

**Comment by levitte** [comment]
> Could you please amend the commit message to describe the bug being fixed.

Sure.

**Comment by levitte** [comment]
New commit message is:

```
SSL: Document SSL_add_{file,dir,store}_cert_subjects_to_stack()

This also removes the incorrect documentation comments by those
functions, and fixes a bug in SSL_add_store_cert_subjects_to_stack(),
where the condition for recursive addition was 'depth == 0' when it
should be 'depth > 0'.
```

**Comment by levitte** [comment]
Merged.

ee669781d5de4c017595149c6036ca24b35aa2c7 SSL: Document SSL_add_{file,dir,store}_cert_subjects_to_stack()

**Pull request closed without merging**